src/test/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/data/matrix_build.xml

```diff
@@ -86,9 +86,6 @@
         </hudson.model.Cause_-UpstreamCause>
       </causes>
     </hudson.model.CauseAction>
```

Comment:
Normally removing xml data from test cases _(especially back compat test cases)_ would defeat the purpose of the test as is normally tests that loading of the data works, and if it doesn't code should be fixed and not data.

In this case its probably OK as the Action probably shouldn't be there due to the veto extension point. But I'll need to dig into the tests a bit first before I can say for sure.